components/scream/src/physics/shoc/shoc_pblintd_init_pot.cpp

```diff
@@ -0,0 +1,16 @@
+#include "shoc_pblintd_init_pot_impl.hpp"
+#include "ekat/scream_types.hpp"
+
+namespace scream {
+namespace shoc {
+
+/*
+ *  * Explicit instantiation for doing ice melting on Reals using the
```

Comment:
Delete "ice melting" to make this generic.

Reply:
yes, done!